code/modules/surgery/tools.dm

```diff
@@ -97,4 +97,57 @@
 
 /obj/item/weapon/surgical_drapes/attack(mob/living/M, mob/user)
 	if(!attempt_initiate_surgery(src, M, user))
-		..()
\ No newline at end of file
+		..()
+		
+/obj/item/weapon/organ_storage //allows medical cyborgs to manipulate organs without hands
+	name = "organ storage bag"
+	desc = "A container for holding body parts."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "evidenceobj"
+
+/obj/item/weapon/organ_storage/afterattack(obj/item/I, mob/user, proximity)
+	if(!proximity)
+		return
+	if(contents.len)
+		to_chat(user, "<span class='notice'>[src] already has something inside it.</span>")
+		return
+	if(isorgan(I))
+		var/obj/item/organ/O = I
+		if(O.status != ORGAN_ORGANIC)

```

Comment:
Does this prevent things like augmentation surgery or cybernetic implant installations?